lib/include/rocRoller/Utilities/Settings.hpp

```diff
@@ -34,6 +34,7 @@
 #include <shared_mutex>
 #include <string>
 #include <vector>
+#include <thread>
```

Comment:
It feels like this shouldn't be needed here. Was this failing in the SettingsTests? If so, I would suggest adding this include statement here: https://github.com/ROCm/rocRoller/blob/main/test/catch/SettingsTest.cpp#L30